CHANGELOG.md

```diff
@@ -1,3 +1,38 @@
+# Release 0.18.4
```

Comment:
0.18.3 hasn't been released yet, so this should go in the 0.18.3 section

Reply:
done